pkg/upgrade/upgrade.go

```diff
@@ -55,7 +55,7 @@ type ResourceSpec struct {
 // CreateDefaultDSC creates a default instance of DSC.
 // Note: When the platform is not Managed, and a DSC instance already exists, the function doesn't re-create/update the resource.
 func CreateDefaultDSC(ctx context.Context, cli client.Client) error {
-	// Set the default DSC name depending on the platform
+	// Set the default DSCC name depending on the platform
```

Comment:
```suggestion
	// Set the default DSC name depending on the platform
```

Not quite sure why this is here :)